tensorflow_lite_support/examples/task/text/desktop/python/nl_classifier_demo.py

```diff
@@ -0,0 +1,41 @@
+import subprocess
+import argparse
+import os
+
+def classify(model_path, text):
+    """Classifies input text into different categories
+
+    Initially the function will download the model and then runs the detection tool.
+
+    Args:
+        model_path: path to model
+        text: input text
+
+    Returns:
+        void
+        
+    """
+    # Run the detection tool:
+    subprocess.run(["bazel run -c opt  tensorflow_lite_support/examples/task/text/desktop:nl_classifier_demo --  --model_path=" + model_path + "  --text='" + text + "'"], shell=True)
+
+
+def main():
+    # Create the parser 
+    parser = argparse.ArgumentParser()
```

Comment:
Please use absl flags to parse the arguments. And then you'll use app.run(main) to run the app. See https://abseil.io/docs/python/guides/app.

Reply:
Moved from argparse to absl flags